core/src/Platforms/iOS/SystemWebview/SystemWebUI.cs

```diff
@@ -66,51 +66,51 @@ public void Authenticate(Uri authorizationUri, Uri redirectUri, RequestContext r
         {
             try
             {
-                if (UIDevice.CurrentDevice.CheckSystemVersion(12, 0))
-                {
-                    asWebAuthenticationSession = new AuthenticationServices.ASWebAuthenticationSession(new NSUrl(authorizationUri.AbsoluteUri),
-                        redirectUri.Scheme, (callbackUrl, error) =>
-                        {
-                            if (error != null)
-                            {
-                                ProcessCompletionHandlerError(error);
-                            }
-                            else
-                            {
-                                ContinueAuthentication(callbackUrl.ToString());
-                            }
-                        });
+                //if (UIDevice.CurrentDevice.CheckSystemVersion(12, 0))
```

Comment:
> //if (UIDevice.CurrentDevice.CheckSystemVersion(12, 0)) [](start = 15, length = 56)

Why is this commented out? We need to support different iOS versions...ASWebAuthSession is for 12+, SFAuthSession is for 11 (now deprecated) and SFSafariViewController is for everything else. this is for system browser, and to be on-par with iOS team, so it can't be commented out. let's discuss other options.

Reply:
These are left over changes from testing. will be reverted